modules/prediction/evaluator/vehicle/junction_mlp_evaluator.h

```diff
@@ -100,9 +100,9 @@ class JunctionMLPEvaluator : public Evaluator {
       const std::vector<double>& feature_values);
 
  private:
-  static const size_t OBSTACLE_FEATURE_SIZE = 3;
+  static const size_t OBSTACLE_FEATURE_SIZE = 4 + 2 * 5;
```

Comment:
Better to have some comments to explain these magic numbers, or refer to some document.